web/test/step_definitions/browser.js

```diff
@@ -1,11 +1,16 @@
 import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
 
-const url = 'http://localhost:8080';
+const url = 'http://localhost:4200';
 
 Given(/^the "([^"]*)" page is open$/, (path) => {
     cy.visit(`${url}/${path}`);
 });
 
 Then(/^the title on the page says "([^"]*)"$/, (check) => {
```

Comment:
Can `check` be `text` to be consistent with the other step def

Reply:
Actually its verifying the title on  the page not the element

Reply:
I mean for the variable name, it is checking for text, so I think it would be more consistent to have use `text` instead of `check` for the variable name

Reply:
okay gotcha !! will fix it too :) Thank you for clarifying 👍